internal/js/modules/k6/browser/common/page_options.go

```diff
@@ -33,6 +33,12 @@ type PageScreenshotOptions struct {
 	Quality        int64          `json:"quality"`
 }
 
+// PageGoBackForwardOptions are options for Page.GoBack and Page.GoForward.
+type PageGoBackForwardOptions struct {
+	WaitUntil LifecycleEvent `json:"waitUntil" js:"waitUntil"`
```

Comment:
Do we need `js:"..."` here? What's the reason `WaitUntil` has it but not `Timeout`?